src/themes/schulportal/login/messages/messages_de.properties

```
@@ -13,4 +13,4 @@ setPassword=Passwort festlegen
 changePassword=Passwort ändern
 updatePasswordTitle=Passwort festlegen
 usernameOrEmail=Benutzername
-accountDisabledMessage=Ihr Benutzerkonto ist gesperrt. Bitte wenden Sie sich an den schulischen Admin.
\ No newline at end of file
+accountDisabledMessage=Bitte wenden Sie sich an Ihren schulischen Administrator/Ihre schulische Administratorin
```

```suggestion
accountDisabledMessage=Ihr Benutzerkonto ist gesperrt. Bitte wenden Sie sich an Ihren schulischen Administrator/Ihre schulische Administratorin.
```
Ich verstehe das Ticket so, dass nur der zweite Satz geändert werden soll.